src/main/java/at/hannibal2/skyhanni/config/Features.java

```diff
@@ -105,7 +105,7 @@ public String getTitle() {
     public DungeonConfig dungeon = new DungeonConfig();
 
     @Expose
-    @Category(name = "Events", desc = "Stuff that is not always available.")
+    @Category(name = "ScoreboardEvents", desc = "Stuff that is not always available.")
```

Comment:
I don't think this was supposed to change -- this is the actual Events Config,

Reply:
Oh ty, yeah it wasnt supposed to